integration-tests/src/test/java/tachyon/client/TachyonFSIntegrationTestIso.java

```diff
@@ -145,9 +156,13 @@ public void lockBlockTest3() throws IOException {
     for (int k = 0; k < numOfFiles; k ++) {
       tFile = mTfs.getFile(fileIds.get(k));
       Assert.assertTrue(tFile.isInMemory());
+      is = tFile.getInStream(ReadType.CACHE);
+      buf = ByteBuffer.allocate((int) tFile.getBlockSizeByte());
+      is.read(buf.array());
       if (k < numOfFiles - 1) {
-        Assert.assertNotNull(tFile.readByteBuffer(0));
+        Assert.assertNotNull(buf);
```

Comment:
Like above, this assert is testing the ByteBuffer.allocate, not the read().

Throughout this test file, the asserts are checking if the bytebuffer was allocated, but not if the read succeeded.